### DISCUSSION TITLE
unblock website with adBlock filters ?

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1081] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
On adblock settings in the filter list tab we can add filters from a custom url.
I'm asking if we can run script with them or unblock websites.
I also discovered that with adblock filters we can add button that can run scripts on the adBlock pop-up like on [youtube](YouTube.com).

**Comment by SIMONHSCH** [comment]
I'm asking that because my school doesn't allow uBlock.

**Reply by SIMONHSCH** [reply]
what are some ways to unblock websites with it ?

**Reply by SIMONHSCH** [reply]
we can ask to someone experimented for that

**Comment by S-PScripts** [comment]
get uBlock on another computer + try there
